core/aws-core/src/main/java/software/amazon/awssdk/awscore/internal/AwsErrorCode.java

```diff
@@ -45,6 +45,7 @@
         throttlingErrorCodes.add("RequestThrottled");
         throttlingErrorCodes.add("RequestThrottledException");
         throttlingErrorCodes.add("EC2ThrottledException");
+        throttlingErrorCodes.add("TransactionInProgressException");
```

Comment:
Checking the list of error code in the SEP, there are also the `LimitExceededException` and `PriorRequestNotComplete` errors. Do we also need to take care of those?

Reply:
`LimitExceededException` actually got removed in #1737.

`PriorRequestNotComplete` is in the retryableErrorCodes list for some reason, so it might introduce breaking change if we move it to throttlingErrorCodes